orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -103,6 +103,7 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 		inferredFile.createNewFile();
 		ReactionInferrer.setEligibleFilename(eligibleFilename);
 		ReactionInferrer.setInferredFilename(inferredFilename);
+		StableIdentifierGenerator.setSpeciesAbbreviation((String) speciesObject.get("abbreviation"));
```

Comment:
Is there a `ReactomeJavaConstants` constant for "abbreviation"? If not, maybe one should be added to `ReactomeJavaConstants`.

Reply:
Yea there isn't. In the old code it extracted the abbreviation from the species name ('H'omo 'SA'piens; 'M'us 'MU'sculus) using a regex. I figured having it in a config made more sense

Reply:
Ok, I think `ReactomeJavaConstants` comes from the code in the CuratorTool that gets exported to reactome.jar or reactome-base.jar. We can look into adding a constant for the value "abbreviation".

Reply:
There are a few constants for newer attributes that could be added to `ReactomeJavaConstants`.  It might be good to look over the current db schema at some point and make sure we have all classes and attributes covered.